README.rst

```diff
@@ -1,30 +1,50 @@
 Table of Contents
------------------
+=================
```

Comment:
Is this file generated? Some of the formatting is really bad. Prefer markdown.

Reply:
Yes this file is generated for the Pypi website where the package is registered to be downloaded via pip and it doesn't accept any kind of format appart for .rst so I made it just for the Pypi website